geotrek/flatpages/locale/fr/LC_MESSAGES/django.po

```diff
@@ -104,14 +104,28 @@ msgstr "Intitulé de l'encart"
 msgid "ID list (separated by comma)"
 msgstr "Liste d'ID (séparés par des virgules)"
 
+msgid "Trek"
+msgstr ""
+
+#, fuzzy
+#| msgid "HTML content"
+msgid "Touristic content"
+msgstr "Contenu HTML"
+
+msgid "Touristic event"
+msgstr ""
+
+msgid "Outdoor Site"
+msgstr ""
```

Comment:
There's something I don't understand. 
Today, the translations are correct without adding "Trek, Touristic event, etc" to these `.po` files. 
I assumed this was defined elsewhere (it seems legitimate because these locales are literally everywhere in this project).

Don't hesitate to tell me i'm wrong. :D

Reply:
The locales are generated by Django application, they were probably new in this app (`flatpages` folder) even though they already were in other apps, and so were already found on the front